flight_planning/v1/flight_planning.yaml

```diff
@@ -116,60 +132,91 @@ components:
             - Nominal
             - OffNominal
             - Contingent
-        nominal_area:
+        area:
           description: >-
-            User nominally intends or intended to fly in this entire area.  If authorizations are required and cannot be
-            granted to cover this entire area, the plan should be rejected.
+            User intends to or may fly anywhere in this entire area.
           type: array
           items:
-            $ref: './astm_f3548_21.yaml#/components/schemas/Volume4D'
-          default: []
-        off_nominal_area:
-          description: >-
-            While experiencing an off-nominal situation, the user's aircraft may travel anywhere in this area.  If the
-            flight_state is nominal, this field must be empty or omitted.
-          type: array
-          items:
-            $ref: './astm_f3548_21.yaml#/components/schemas/Volume4D'
+            $ref: './geotemporal.yaml#/components/schemas/Volume4D'
           default: []
 
     UpsertFlightPlanRequest:
+      description: >-
+        Client request to emulate a user performing a flight planning action.  Note that the USS should consider the
+        user to be actually performing (or attempting to perform) this action, and should not consider the user to be
+        merely exploring the possibility of performing the action, or merely requesting to perform the action.
       type: object
       required:
-        - intended_flight
+        - flight
```

Comment:
```suggestion
        - flight_plan
```

Reply:
Thanks, fixed.  I am not a fan of the OpenAPI style for required fields :/